internal/resolution/manifest/maven.go

```diff
@@ -173,57 +257,6 @@ func mavenOrigin(list ...string) string {
 	return result
 }
 
-func makeMavenDepType(dependency maven.Dependency, origin string) dep.Type {
```

Comment:
Just to check, this is removed because the logic has been moved to resolve right?

Reply:
this is now in `deps.dev/util/resolve`, so yes if this is what you refer to by `resolve`